### ISSUE TITLE
Booster Fonts Fallback not working as expected - undeclared collection of fonts injected instead?

### ISSUE THREAD

**Issue by sbastias** [basics/nuxt-booster#1146]
**Describe the bug**
Font fallback is not wokring as expected.

**To Reproduce**
```
// nuxt.config.js

fonts: [
    //@ts-ignore
    {
      family: 'Merriweather',
      locals: ['Merriweather'],
      fallback: ['Times New Roman', 'Times', 'Georgia', 'serif'],
      variances: [
        {
          style: 'normal',
          weight: 400,
          sources: [
            { src: '@/assets/fonts/Merriweather/Merriweather-Regular.woff2', type:'woff2' },
            { src: '@/assets/fonts/Merriweather/Merriweather-Regular.woff', type:'woff' },
            { src: '@/assets/fonts/Merriweather/Merriweather-Regular.ttf', type:'ttf' },
          ]
        },
        {
          style: 'italic',
          weight: 400,
          sources: [
            { src: '@/assets/fonts/Merriweather/Merriweather-Italic.woff2', type:'woff2' },
            { src: '@/assets/fonts/Merriweather/Merriweather-Italic.woff', type:'woff' },
            { src: '@/assets/fonts/Merriweather/Merriweather-Italic.ttf', type:'ttf' },
          ]
        },
        {
          style: 'normal',
          weight: 900,
          sources: [
            { src: '@/assets/fonts/Merriweather/Merriweather-Black.woff2', type:'woff2' },
            { src: '@/assets/fonts/Merriweather/Merriweather-Black.woff', type:'woff' },
            { src: '@/assets/fonts/Merriweather/Merriweather-Black.ttf', type:'ttf' },
          ]
        },
      ]
    },
...
```

**Expected behavior**
Fallback should be loaded in rendered site code.

**Screenshots**
The injected fallback styles do not include the specified fallback options:
<img width="653" alt="image" src="https://github.com/user-attachments/assets/718d8f57-337f-4f1d-b86d-726a8b0fd109">

**Additional context**
You can see FOUC happening here (screen recording)
https://capture.dropbox.com/8mc2JaMiAobPZhWV

**Comment by sbastias** [comment]
It looks like nuxt-fontaine is the culprit here:
https://github.com/nuxt-modules/fontaine

I set `disableNuxtFontaine: true` in nuxt.config.js and it's working as expected.

This is a bug - the declared fallback should be passed to nuxt-fontaine.

**Comment by ThornWalli** [comment]
Hello @sbastias, 

The fallbacks are intended for the `font-family` in the client.

Have you set Stage (Hero) to `critical`, if not the fonts will be reloaded.

If the `v-font` is installed in a subcomponent, a `useBoosterCritical` must be defined on the main component.

If you have the `v-font` directly in the component, you can use the component as an example. https://github.com/basics/nuxt-booster/blob/main/playground/components/modules/Stage.vue

It is important that there is a `critical` attribute on the main component.

https://github.com/basics/nuxt-booster/blob/9d8b38cc11bec32ff339b2c63e4e4697256546aa/playground/pages/index.vue#L1-L35

Important: critical only receives the components that are visible in the initial viewport.

**Comment by sbastias** [comment]
Ok, in the section component, I declared `const {$getFont} = useBoosterFonts({critical: true})` and  I added `v-font="$getFont('Merriweather', 900)` directly on the H1 for the heroes.

I reinstated Fontaine and it works - thank you! 

But I would suggest explaining 'critical' overrides the Fontaine default fallbacks - it was really confusing. Maybe in the config for the module, a comment `fallback: [....] // only for critical components`
